main.py

```diff
@@ -1,47 +1,42 @@
+from typing import Dict
+from models import CountryVaccineSummary
 from fastapi import FastAPI
 from datawork import getCountryVaccinePercentages
-from fastapi.responses import HTMLResponse
-from fastapi.responses import JSONResponse
 
-import json
 
 app = FastAPI()
 
-@app.get("/data")
+@app.get("/data", response_model=CountryVaccineSummary)
 async def root():
     data = getCountryVaccinePercentages()
     return data
 
 @app.get("/mostpopular")
-async def root():
+async def most_popular() -> Dict[str, str]:
```

Comment:
FastAPI works better when the response types are provided, also the routes had the same names, which could cause issues.